meta.go

```diff
@@ -32,8 +32,66 @@ func (m Meta) String() string {
 	return s.String()
 }
 
+func (m Meta) Equals(b Meta) bool {
+	akeys := m.Keys()
+	bkeys := b.Keys()
+
+	if len(akeys) != len(bkeys) {
+		return false
+	}
+
+	sort.Strings(akeys)
+	sort.Strings(bkeys)
```

Comment:
Why pulling and sorting the keys? They don't seem to be used at all. The same could be achieved in a more performant way like this:

```suggestion
	if len(m) != len(b) {
		return false
	}
```

Reply:
My bad, I missed something here. I should be iterating over `akeys` to see if `bkeys` exist in the reference map and its value is equivalent. I'll fix this up.

Reply:
Well, there's no need to sort the keys for that either. Just ranging on `m` will test that `b` indeed has the same keys and value.

Reply:
True, I wonder what I was thinking here 🙃 I'll try remind myself when I come back to this.